fsspec/tests/test_utils.py

```diff
@@ -381,7 +381,7 @@ def client(self):
             return mock
 
         def func_2(self):
-            assert False, "have to overwrite this"
+            assert False, "have to overwrite this"  # noqa: B011
```

Comment:
`raise AssertError` ?

Reply:
Or perhaps disable this rule in all tests?

Reply:
This is certainly a special case, it's not normal to `assert False`! On second thoughts, I'm OK with the original `noqa` too.

Reply:
I changed to `raise AssertionError` in the mean time. Is that OK?